clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -673,9 +673,6 @@ float dot(float3, float3);
 _HLSL_BUILTIN_ALIAS(__builtin_hlsl_dot)
 float dot(float4, float4);
 
-_HLSL_BUILTIN_ALIAS(__builtin_hlsl_dot)
-double dot(double, double);
-
```

Comment:
Surprised to see this here based on the description. Wonder if we could hand hold readers a bit more to explain why this was removed?

Reply:
Oh, I think this can stay in once this PR lands: https://github.com/llvm/llvm-project/pull/106811